app/views/podcast_episodes/show.html.erb

```diff
@@ -28,7 +28,7 @@
 
 <% end %>
 
-<div class="podcast-episode-container">
+<div class="podcast-episode-container" data-meta='{"podcastName":"<%= @podcast.title %>","episodeName":"<%= @episode.title %>","podcastImageUrl":"<%= "#{app_url(@podcast.image_url)}" %>"}'>
```

Comment:
Is it possible that this will cause a problem if `@episode.title` includes a `'`?

Not sure I'm wrappimg my head around it the right way, but maybe we need to do something like creating a hash and then `.to_json` (I think this would mean the outer quotes should be double-quoted?)

Reply:
I don't think `'` would be a problem since it's wrapped by double quotes in the data attribute, but I also hadn't thought about using `to_json`. So now that you bring it up I do consider `to_json` a safer approach in general (for escaping special characters all around).

I do think outer quotes should wrap the content using doubles `"` in that case because `to_json` doesn't escape single quotes. I'll push the changes in soon 👍🏼